src/types/User.ts

```diff
@@ -1,4 +1,19 @@
-export interface Experience {
+export enum PlaceInTalkShowStudioVenue {
+  stage = "stage",
+  audience = "audience",
+  requesting = "requesting",
+}
+
+export interface TalkShowStudioExperience {
+  place?: PlaceInTalkShowStudioVenue;
+  isSharingScreen?: boolean;
+  isMuted?: boolean;
+  isUserCameraOff?: boolean;
+  cameraClientUid?: string;
+  screenClientUid?: string;
+}
+
+export interface Experience extends TalkShowStudioExperience {
```

Comment:
This seems... incorrect? If anything, the `TalkShowStudioExperience` should be extending from `Experience` I would expect?